config/ndlar/ndlar_full_chain_flash_nersc_240819.cfg

```diff
@@ -0,0 +1,499 @@
+        node_feats: 33 # 16 (geo) + 3 (extra) + 6 (points) + 6 (directions) + 2 (local dedxs)
+        edge_feats: 19
+        node_pred:
+          type: 6
+          primary: 2
+          orient: 2
+          #momentum: 1
+          #vertex: 5
+        edge_pred: 2
+        edge_layer:
+          name: mlp
+          mlp:
+            depth: 3
+            width: 128
+            activation:
+              name: lrelu
+              negative_slope: 0.1
+            normalization: batch_norm
+        node_layer:
+          name: mlp
+          reduction: max
+          attention: false
+          message_mlp:
+            depth: 3
+            width: 128
+            activation:
+              name: lrelu
+              negative_slope: 0.1
+            normalization: batch_norm
+          aggr_mlp:
+            depth: 3
+            width: 128
+            activation:
+              name: lrelu
+              negative_slope: 0.1
+            normalization: batch_norm
+
+    grappa_inter_loss:
+      node_loss:
+        type:
+          name: class
+          target: pid
+          loss: ce
+          balance_loss: true
+        primary:
+          name: class
+          target: inter_primary
+          loss: ce
+          balance_loss: true
+        orient:
+          name: orient
+          loss: ce
+        #momentum:
+        #  name: reg
+        #  target: momentum
+        #  loss: berhu
+        #vertex:
+        #  name: vertex
+        #  primary_loss: ce
+        #  balance_primary_loss: true
+        #  regression_loss: mse
+        #  only_contained: true
+        #  normalize_positions: true
+        #  use_anchor_points: true
+        #  return_vertex_labels: true
+        #  detector: icarus
+      edge_loss:
+        name: channel
+        target: interaction
+
+# Build output representations
+build:
+  mode: both
+  units: cm
+  fragments: false
+  particles: true
+  interactions: true
+  
+# Run post-processors
+post:
+  shape_logic:
+    enforce_pid: true
+    enforce_primary: true
+    priority: 3
+  #particle_threshold:
+  #  track_pid_thresholds:
+  #    4: 0.85
+  #    2: 0.1
+  #    3: 0.5
+  #    5: 0.0
+  #  shower_pid_thresholds:
+  #    0: 0.5 
+  #    1: 0.0
+  #  primary_threshold: 0.1
+  #  priority: 2
+  #track_extrema:
+  #  method: gradient
+  #  priority: 2
+  direction:
+    obj_type: particle
+    optimize: true
+    run_mode: both
+    priority: 1
+  calo_ke:
+    run_mode: reco
+    scaling: 1.
+    shower_fudge: 1/0.83
+    priority: 1
+  csda_ke:
+    run_mode: reco
+    tracking_mode: step_next
+    segment_length: 5.0
+    priority: 1
+  mcs_ke:
+    run_mode: reco
+    tracking_mode: bin_pca
+    segment_length: 5.0
+    priority: 1
+  topology_threshold:
+    ke_thresholds:
+      4: 50
+      default: 25
+  vertex:
+    use_primaries: true
+    update_primaries: false
+    priority: 1
+  containment:
+    detector: 2x2
+    margin: 5.0
+    mode: detector
+  fiducial:
+    detector: 2x2
```

Comment:
Same comment, switch to `dunend`

Reply:
Fixed in 2916c14ad15e2d9f030c4629d2cfd08bdf00ebc8